.cirrus.yml

```diff
@@ -114,8 +114,7 @@ task:
         SCRIPTS: scripts/windows*
 
       matrix:
-        - name: windows-ci-vs-2019
-        - name: windows-ci-mingw64
+        - name: windows
```

Comment:
I think it should still have -ci in the name.  We're going to have to update the CI tasks in postgres quickly afterwards...